content/en/database_monitoring/guide/sql_deadlock.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Configuring Deadlock monitoring on SQL Server
+further_reading:
+- link: "/database_monitoring/"
+  tag: "Documentation"
+  text: "Database Monitoring"
+- link: "/database_monitoring/setup_sql_server/"
+  tag: "Documentation"
+  text: "Setting Up SQL Server"
+- link: "/database_monitoring/troubleshooting/"
+  tag: "Documentation"
+  text: "Troubleshooting Database Monitoring"
+---
+
+The Deadlock view allows you to explore deadlocks events in your SQL Server database.
+This guide assumes you have configured Database Monitoring for your [SQL Server][1].
```

Comment:
```suggestion
The Deadlock view enables you to explore deadlock events in your SQL Server database. A deadlock occurs when two or more processes are unable to proceed because each is waiting for the other to release resources. This guide assumes that Database Monitoring has been configured for your [SQL Server][1].
```

Reply:
Added a description of what a "deadlock" is to the introduction. Let me know if the description is accurate, or if it needs to be revised.